Code_Exercises/CMakeLists.txt

```diff
@@ -9,28 +9,28 @@
 ]]
 
 # CMake function to build ab AdaptiveCpp configuration and create executable binaries
-# from AdaptiveCpp libraries.
```

Comment:
I think the changes in this file have conflicted with #335, perhaps the only change should be adding line 68 (i.e. `add_subdirectory(Functors)`).